var/spack/repos/builtin/packages/tasmanian/package.py

```diff
@@ -158,37 +158,20 @@ def cmake_args(self):
 
         return args
 
-    # TODO: Replace this method and its 'get' use for cmake path with
-    #   join_path(self.spec['cmake'].prefix.bin, 'cmake') once stand-alone
-    #   tests can access build dependencies through self.spec['cmake'].
-    def cmake_bin(self, set=True):
-        """(Hack) Set/get cmake dependency path."""
-        filepath = join_path(self.install_test_root, "cmake_bin_path.txt")
-        if set:
-            with open(filepath, "w") as out_file:
-                cmake_bin = join_path(self.spec["cmake"].prefix.bin, "cmake")
-                out_file.write("{0}\n".format(cmake_bin))
-        elif os.path.isfile(filepath):
-            with open(filepath, "r") as in_file:
-                return in_file.read().strip()
-
     @run_after("install")
     def setup_smoke_test(self):
         if not self.spec["cmake"].satisfies("@3.10:"):
-            tty.msg("Error tasmanian test: CMake 3.10 or higher is required")
+            tty.warn("CMake 3.10 or higher is required for stand-alone tests")
```

Comment:
Since Tasmanian requires CMake 3.10 to build, do we need to check this at all?

Reply:
Good point.  I can remove this check.